### PR TITLE
Update api3 explorer url path for consistency

### PR DESCRIPTION
**Pull request by colemanw**
Overview
----------------------------------------
For consistency with the api4 explorer let's include the version in the url.

Before
----------------------------------------
Api3 explorer at `civicrm/api`.

After
----------------------------------------
Api3 explorer at `civicrm/api3`. Redirects from `civicrm/api`.

### CODE REVIEW

**Comment by MikeyMJCO** [comment]
I'm not seeing where the redirect is defined from reading the code on this one?

File: CRM/Core/xml/Menu/Misc.xml [hunk 1]
```diff
@@ -85,6 +85,12 @@
   </item>
   <item>
      <path>civicrm/api</path>
+     <page_callback>CRM_Core_Page_Redirect</page_callback>
+     <page_arguments>url=civicrm/api3</page_arguments>
+     <access_arguments>access CiviCRM</access_arguments>
```
Comment (colemanw) [inline CRM/Core/xml/Menu/Misc.xml]: @MikeyMJCO this adds the redirect.

Comment (MikeyMJCO) [reply CRM/Core/xml/Menu/Misc.xml]: Thanks :-D missed that!

Comment (MikeyMJCO) [reply CRM/Core/xml/Menu/Misc.xml]: I'll do an r-run on this shortly.

Comment (MikeyMJCO) [reply CRM/Core/xml/Menu/Misc.xml]: This runs fine locally - both URLs work.

**Comment by MikeyMJCO** [comment]
* General standards
    * ([`r-explain`](https://docs.civicrm.org/dev/en/latest/standards/review/#r-explain)) __Pass__
    * ([`r-user`](https://docs.civicrm.org/dev/en/latest/standards/review/#r-user)) __Pass__
    * ([`r-doc`](https://docs.civicrm.org/dev/en/latest/standards/review/#r-doc)) __Pass__
    * ([`r-run`](https://docs.civicrm.org/dev/en/latest/standards/review/#r-run)) __Pass__
* Developer standards
    * ([`r-tech`](https://docs.civicrm.org/dev/en/latest/standards/review/#r-tech)) __Pass__
    * ([`r-code`](https://docs.civicrm.org/dev/en/latest/standards/review/#r-code)) __Pass__
    * ([`r-maint`](https://docs.civicrm.org/dev/en/latest/standards/review/#r-maint)) __Pass__
    * ([`r-test`](https://docs.civicrm.org/dev/en/latest/standards/review/#r-test)) __Pass__

**Comment by seamuslee001** [comment]
thanks @colemanw @MikeyMJCO

**Pull request merged**